pkg/vm/opcode/opcode.go

```diff
@@ -65,31 +65,36 @@ const (
 	CALL      Opcode = 0x34
 	CALLL     Opcode = 0x35
 
-	// Legacy
-	OLDPUSH1 Opcode = 0x51
+	// Stack
+	DEPTH Opcode = 0x43
+	DROP  Opcode = 0x45
+	NIP   Opcode = 0x46
+	XDROP Opcode = 0x48
+	// CLEAR    Opcode = 0x49
+	DUP      Opcode = 0x4A
+	OVER     Opcode = 0x4B
+	PICK     Opcode = 0x4D
+	TUCK     Opcode = 0x4E
+	SWAP     Opcode = 0x50
+	OLDPUSH1 Opcode = 0x51 // FIXME remove #xxx
+	ROT      Opcode = 0x7B
```

Comment:
I think we can make `OLDPUSH` an alias of new `ROT` which is `0x51` and then make a kludge in `Context` to treat one-byte script containing `ROT` as `PUSH1`. This way we won't have to update scripts again after `OLDPUSH1` removal.

Reply:
Yeah, it is actually sufficient.